modules/services/cloud-connector-ecs/ecs-service.tf

```diff
@@ -16,6 +16,10 @@ resource "aws_ecs_service" "service" {
   task_definition       = aws_ecs_task_definition.task_definition.arn
   wait_for_steady_state = true
   tags                  = var.tags
+
+  lifecycle {
+    ignore_changes = [desired_count]
```

Comment:
We need to ignore the desired count because will be modified by the autoscaling and further `terraform apply` executions will modify this back to `1`.